docs/development.md

```diff
@@ -316,12 +316,15 @@ therefore delivery of emails is not tested when testing against the dev server.
 
 > Client scripts are scripts that remotely manipulate data on GAE via its Remote API. They are run as standard Java applications.
 
-Most of developers may not need to write and/or run client scripts but if you are to do so *in a production environment*, additional steps are required:
+Most of developers may not need to write and/or run client scripts but if you are to do so, take note of the following:
 
-1. Download and install Google Cloud SDK from [here](https://cloud.google.com/sdk/downloads).
-1. Run `gcloud auth login` and choose your Google account for authentication.
+* In order to run any script *in a production environment*, you need to authorize your account for [Application Default Credentials](https://developers.google.com/identity/protocols/application-default-credentials).
+  ```sh
+  gcloud auth application-default login
+  ```
+  Follow the steps until you see `Credentials saved to file: [...].` printed on the console.
 
-You are now ready to run your scripts.
+* Running scripts via command line is not supported.
```

Comment:
Can explain this?

Reply:
Trying to `java DataMigrationScript` (for e.g.) in the built classes `(build/classes/...)` won't work because GAE understandably strips all the unnecessary `main` methods.

The only way then is to manually compile and run but it's a rather tedious task, you have to `javac -cp {huge amount of jars and classes} DataMigrationScript.java` where the JARs and dependent classes are strewn everywhere. Not that it's not doable, but I don't see a feasible way to support it with a Gradle shortcut method, i.e. this is what I mean by "not supported" although it may not be the best way to represent it.

Reply:
Note also that this is not due to the Cloud SDK migration; this has been the case from the start.

Reply:
Oh I see. That's fine then.